app/assets/templates/directives/buddy_request_box.html

```diff
@@ -0,0 +1,17 @@
+<div id="buddyRequestInfoBox"></div>
+<div class="panel panel-info" ng-controller="BuddyNotificationsCtrl as buddyCtrl">
```

Comment:
apply the controller directly to the directive itself, then you don't have to do this, and you don't have to do dom manipulation with jQuery.